core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10031,4 +10031,37 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
 
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBigQuery));
   }
+
+
+  @Test public void testParseDateTimeFormat() {
+    final RelBuilder builder = relBuilder();
+    final RexNode parseTSNode1 = builder.call(SqlLibraryOperators.PARSE_DATE,
+        builder.literal("YYYYMMDD"), builder.literal("99991231"));
```

Comment:
Add this too:
final RexNode parseDateNode2 = builder.call(SqlLibraryOperators.PARSE_DATE,
        builder.literal("YYYY MM DD"), builder.literal("1998 05 12"));

Reply:
Done